packages/bpk-foundations-common/app/typography.json

```diff
@@ -0,0 +1,35 @@
+{
+  "aliases": {
+    "FONT_SIZE_CAPS": 10,
+    "FONT_SIZE_XS": 12,
+    "FONT_SIZE_SM": 14,
+    "FONT_SIZE_BASE": 16,
+    "FONT_SIZE_LG": 20,
+    "FONT_SIZE_XL": 24,
+    "FONT_SIZE_XXL": 30,
+    "FONT_SIZE_XXXL": 36,
+    "FONT_SIZE_XXXXL": 48,
+    "FONT_SIZE_5XL": 64,
+    "FONT_SIZE_6XL": 76,
+    "FONT_SIZE_7XL": 96,
+    "FONT_SIZE_8XL": 120,
+    "LINE_HEIGHT_XS": 14,
+    "LINE_HEIGHT_SM": 16,
+    "LINE_HEIGHT_BASE": 19,
+    "LINE_HEIGHT_BASE_TIGHT": 16,
+    "LINE_HEIGHT_LG": 23,
+    "LINE_HEIGHT_LG_TIGHT": 20,
+    "LINE_HEIGHT_XL": 28,
+    "LINE_HEIGHT_XL_TIGHT": 24,
+    "LINE_HEIGHT_XXL": 42,
+    "LINE_HEIGHT_XXXL": 40,
+    "LINE_HEIGHT_XXXXL": 48,
+    "LINE_HEIGHT_5XL": 64,
+    "LINE_HEIGHT_6XL": 76,
+    "LINE_HEIGHT_7XL": 96,
+    "LINE_HEIGHT_8XL": 120,
+    "LETTER_SPACING_TIGHT": -0.02,
+    "FONT_WEIGHT_BOOK": "400",
+    "FONT_WEIGHT_BOLD": "700"
```

Comment:
I think for now we also need the 900 version which is the black (which confusingly is used as `FONT_WEIGHT_BOLD` on iOS in the past). right now we merged what was semibold + bold to the same styles, but they're different.